experimental/AlgebraicStatistics/src/DiscreteGraphicalModels.jl

```diff
@@ -0,0 +1,86 @@
+function bron_kerbosch(G, R, P, X)
+
+    cliques = []
```

Comment:
Note that this will be a `Vector{Any}`, leading to type unstable code down the line. If you have any information about what the content might be, you could let Julia know: If `T` is a supertype of the content, you can use
```suggestion
    cliques = T[]
```
to get an empty `Vector{T}` here.